platformLibs/src/platform/ios/builtin.def

```diff
@@ -2,6 +2,9 @@ depends = posix
 package = platform.builtin
 headerFilter = **
 language = C
+
+excludedFunctions.ios_arm32 = builtin_clear_cache
+excludedFunctions.ios_arm64 = builtin_clear_cache
```

Comment:
What are these ones?
Which binary contains definitions of these functions on ios_x64?

Reply:
This function is defined below in the same def file. It calls `__builtin___clear_cache`, which is for some reason missing on ios_armXX.